### PR TITLE
APM > .NET > Add .NET 7 support to docs

### PR DESCRIPTION
**Pull request by zacharycmontoya**
### What does this PR do? What is the motivation?
There are several places in our docs that were not updated to indicate our .NET 7 support. This PR fixes that.

### Merge instructions
- [X] Please merge after reviewing

### Additional notes
N/A

### CODE REVIEW

**Review by drichards-87 — APPROVED** [review]
Look good. I see you added another reviewer though. I'll wait until they approve and then I'll merge the PR.

**Review by tonyredondo — APPROVED** [review]
LGTM

**Pull request merged**
